deployment/terraform/assets/dashboard_data.json

```diff
@@ -1859,5 +2047,6 @@
   "timezone": "browser",
   "title": "Mattermost Load-Test Performance Monitoring",
   "uid": "000000011",
-  "version": 4
+  "version": 3,
```

Comment:
This got set back to 3. I didn't see any problems with this, but lmk if you guys want me to reset it to 4.